lib/stm32/g4/Makefile

```diff
@@ -56,6 +56,7 @@ OBJS += usb_hid.o
 OBJS += usb_midi.o
 OBJS += usb_msc.o
 OBJS += st_usbfs_core.o st_usbfs_v2.o
+OBJS += fdcan.o
```

Comment:
this list is kept sorted, with the USB grouped separately.  This shouldgo up between dmamux.o and flash.o...